contracts/SL2RD.sol

```diff
@@ -300,12 +298,14 @@ contract SL2RD is
     function transferNextAvailable(
         address to_
     ) public onlyOwnerOrOperator nonReentrant {
+        address splitOwner = ownerOf(_nextAvailableCommunitySlot);
+
         // Gates process to only continue until community reservations are depleted.
         require(_nextAvailableCommunitySlot < _totalCommunitySlots, "SHARE031");
 
         // Ensure the initial owner still owns the community slot before transfer.
         require(
-            ownerOf(_nextAvailableCommunitySlot) == _initialOwner.value,
+            splitOwner == _addresses.value[_nextAvailableCommunitySlot],
```

Comment:
@Jpsassine Is there a case in which this could be false?

Reply:
This is checking to make sure that the splitOwner is one of the original participants in the split, since this PR is allowing there to be multiple owners at initialization. 

3 co owners that get 33% rand distro, then they want to give 10% to community. Previously with the _intialOwner set to _addresses[0] only one of these owner's slots would be allowed to be transferred and you simply could not have 3 true initial owners, you would have to trust the true initial owner to transfer 66% to the other two and then decide where to go from there.